feature_creation_scripts/antarctic_ocean_regions/antarctic_ocean_regions.py

```diff
@@ -211,40 +238,73 @@ def split_rectangle(lon0, lon1, lat0, lat1, name, author, tags, fcContour):
     return fc
 
 
+def split_polygon(lon_vector, lat_vector, name, author, tags, fcContour):
+    fc = make_polygon(lon_vector, lat_vector, name, author, tags)
+
+    fcDeep = fc.difference(fcContour)
+
+    props = fcDeep.features[0]['properties']
+    props['name'] = props['name'] + ' Deep'
+    props['tags'] = props['tags'] + ';Deep'
+    props['zmin'] = -1000.
+    props['zmax'] = -400.
+
+    fcShelf = fc.difference(fcDeep)
+
+    props = fcShelf.features[0]['properties']
+    props['name'] = props['name'] + ' Shelf'
+    props['tags'] = props['tags'] + ';Shelf'
+    props['zmin'] = -1000.
+    props['zmax'] = -200.
+
+    fc.merge(fcDeep)
+    fc.merge(fcShelf)
+
+    return fc
+
+
 def main():
-    author = 'Xylar Asay-Davis'
+    xylar = 'Xylar Asay-Davis'
+    carolyn = 'Carolyn Begeman'
     timTags = 'Antarctic;Timmermann'
     orsiTags = 'Antarctic;Orsi'
+    antTag = 'Antarctic'
 
     # make a geometric fieatures object that knows about the geometric data
     # cache up a couple of directories
     gf = GeometricFeatures('../../geometric_data')
 
     bedmap2_bin_to_netcdf('bedmap2.nc')
 
-    fcContour700 = get_longest_contour(contourValue=-700., author=author)
-    fcContour800 = get_longest_contour(contourValue=-800., author=author)
+    fcContour700 = get_longest_contour(contourValue=-700., author=xylar)
+    fcContour800 = get_longest_contour(contourValue=-800., author=xylar)
 
     fc = FeatureCollection()
 
-    fcWeddell = split_rectangle(
-        lon0=-63., lon1=0., lat0=-80., lat1=-65., name='Weddell Sea',
-        author=author, tags=timTags, fcContour=fcContour800)
+    fcWeddell = split_polygon(
+        lon_vector=[-100., -86., -63., -66., -60., -51., 0., 0.],
+        lat_vector=[-81., -75., -73., -67., -64., -62., -62., -90.],
+        name='Weddell Sea',
+        author=carolyn, tags=antTag, fcContour=fcContour800)
 
     # get rid of the Weddell Sea because we're not that happy with this
     # definition, but keep the deep/shelf ones
     fcWeddell.features = fcWeddell.features[1:]
     fc.merge(fcWeddell)
 
-    fcEW = split_rectangle(
-        lon0=-20., lon1=25., lat0=-80., lat1=-55., name='Eastern Weddell Sea',
-        author=author, tags=orsiTags, fcContour=fcContour800)
+    fcEW = split_polygon(
+        lon_vector=[-20., -20., 10., 25., 25.], 
+        lat_vector=[-80., -60., -55., -55., -80.], 
+        name='Eastern Weddell Sea',
+        author=xylar, tags=antTag, fcContour=fcContour800)
 
     fc.merge(fcEW)
 
-    fcWW = split_rectangle(
-        lon0=-63., lon1=-20., lat0=-80., lat1=-60., name='Western Weddell Sea',
-        author=author, tags=orsiTags, fcContour=fcContour800)
+    fcWW = split_polygon(
+        lon_vector=[-100., -86., -63., -66., -60., -51., -20., -20.],
+        lat_vector=[-81., -75., -73., -67., -64., -62., -62., -85.],
```

Comment:
```suggestion
        lat_vector=[-81., -75., -73., -67., -64., -60., -60., -85.],
```
@xylar I think this is more consistent with the Orsi figure.

Reply:
Freel free to commit this.